pkg/lifecycle/unfork/unforker.go

```diff
@@ -287,14 +292,28 @@ func (l *Unforker) generatePatches(fs afero.Afero, step api.Unfork, upstreamMap
 				return errors.Wrap(err, "read forked")
 			}
 
+			forkedResoruce, err := util.NewKubernetesResource(forkedData)
```

Comment:
`Resource`

Reply:
```suggestion
			forkedResource, err := util.NewKubernetesResource(forkedData)
```

Reply:
We have suggested changes, might as well use them!

Reply:
@laverya has to be changed in more than one spot

Reply:
Also no IDE tools to guard against bad syntax

Reply:
Awww

Reply:
(but probably right)